analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -36,6 +38,33 @@ public override bool ShouldExecute()
         return finder.HasPotentialNullableValueAccess;
     }
 
+    protected override ProgramState PreProcessSimple(SymbolicContext context)
+    {
+        var operationInstance = context.Operation.Instance;
+        if (operationInstance.Kind == OperationKindEx.PropertyReference
+            && operationInstance.ToPropertyReference() is { Instance: var instance } propertyReference
+            && propertyReference.Property.Name == nameof(Nullable<int>.Value)
+            && IsNull(instance))
+        {
+            ReportIssue(instance, instance.Syntax.ToString());
+        }
+        else if (operationInstance.Kind == OperationKindEx.Conversion
+            && operationInstance.ToConversion() is { Operand: var operand } conversion
```

Comment:
Same as above
```suggestion
            && operationInstance.ToConversion() is { } conversion
```

Reply:
I use `operand` to access its type, and it's not obvious to me that it can't be `null`.

So I haven't changed it to `operationInstance.ToConversion() is { Operand: { } operand } conversion`.

Reply:
Operations don't have `null` in their basic properties, otherwise they wouldn't be created.

Unless you can make it break with UT (or IT, or Peach validation later), there's no need for null check.